harmony_service/adapter.py

```diff
@@ -145,7 +145,7 @@ def process_item(self, item: Item, source: HarmonySource) -> Item:
 
         except Exception as exception:
             self.logger.exception(exception)
-            raise HyBIGServiceError from exception
+            raise HyBIGServiceError(str(exception)) from exception
```

Comment:
This actually is a change before we logged the correct exception and raised a new one from the old, but the workflow-ui didn't show any information about it.